#[routes] macro requires no_ws_resolvers flag even when WebSocket URLs are unused

## Summary

When using `#[routes]` macro without any arguments on a new REST-only project created with `reinhardt-admin startproject --with-rest`, the compilation fails with:

```
error[E0433]: cannot find `ws_urls` in `urls`
```

## Steps to Reproduce

1. Install CLI: `cargo install reinhardt-admin-cli`
2. Create project: `reinhardt-admin startproject --with-rest myproject .`
3. Run: `cargo check`

## Expected Behavior

A newly scaffolded `--with-rest` project should compile out of the box without requiring additional flags.

## Actual Behavior

Compilation fails because the `#[routes]` macro tries to resolve `ws_urls` module which doesn't exist in a REST-only project.

## Workaround

Adding `server_only, no_ws_resolvers` flags to the macro:

```rust
#[routes(server_only, no_ws_resolvers)]
pub fn routes() -> UnifiedRouter {
    // ...
}
```

## Suggestion

Either:
1. The `--with-rest` template should generate `#[routes(server_only, no_ws_resolvers)]` by default
2. Or the `#[routes]` macro should gracefully skip `ws_urls` resolution when the module doesn't exist

## Environment

- reinhardt-web: 0.1.0
- reinhardt-admin-cli: 0.1.0
- Rust: 1.95.0
- OS: macOS (Darwin 25.4.0)

--- a/crates/reinhardt-commands/templates/project_restful_template/src/config/urls.rs.tpl
+++ b/crates/reinhardt-commands/templates/project_restful_template/src/config/urls.rs.tpl
@@ -5,7 +5,7 @@
 use reinhardt::prelude::*;
 use reinhardt::routes;
 
-#[routes]
+#[routes(server_only, no_ws_resolvers)]
 pub fn routes() -> UnifiedRouter {
     let router = UnifiedRouter::new();
 

